### PR TITLE
Validation Bulk Fixes

### PR DESCRIPTION
**Pull request by mohitp930**

### CODE REVIEW

**Comment by mohitp930** [comment]
Global effort to fix build validation errors

@stevemunk

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

File: BingMaps/rest-services/common-parameters-and-types/type-identifiers/eat-drink-types.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ---
 # required metadata
 title: "Eat Drink Identifiers | Microsoft Docs"
+description: Type IDs for eating and drinking establishments.
```
Comment (stevemunk) [inline BingMaps/rest-services/common-parameters-and-types/type-identifiers/eat-drink-types.md:4]:
```suggestion
description: A list of type identifiers for eating and drinking establishments with descriptions of each type.
```

File: BingMaps/rest-services/common-parameters-and-types/type-identifiers/index.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 ---
 # required metadata
 title: "Type Identifiers | Microsoft Docs"
+description: Learn about top-level IDs used by the Bing Maps APIs.
```
Comment (stevemunk) [inline BingMaps/rest-services/common-parameters-and-types/type-identifiers/index.md:4]:
```suggestion
description: An overview page listing the top-level type identifiers used by the Bing Maps APIs.
```

File: BingMaps/rest-services/common-parameters-and-types/type-identifiers/see-do-types.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
 ---
 # required metadata
 title: "SeeDo Identifiers | Microsoft Docs"
+description: Type IDs for events, activities, parks and recreation, and both natural and man-made landmarks.
```
Comment (stevemunk) [inline BingMaps/rest-services/common-parameters-and-types/type-identifiers/see-do-types.md:4]:
```suggestion
description: A list of type identifiers that define events, activities, parks, and other attractions.
```

File: BingMaps/rest-services/common-parameters-and-types/type-identifiers/shop-types.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
 ---
 # required metadata
 title: "Shop Type Identifiers | Microsoft Docs"
+description: Type IDs for shopping establishments.
```
Comment (stevemunk) [inline BingMaps/rest-services/common-parameters-and-types/type-identifiers/shop-types.md:4]:
```suggestion
description: A list of type identifiers that define various different types of shopping establishments.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by stevemunk** [comment]
@mohitp930 It looks like there is a problem with BingMaps/rest-services/common-parameters-and-types/location-and-recognition-entity-types.md.  Please fix. It looks like there may be an invalid character in the first column of the first row.

image

**Comment by stevemunk** [comment]
@mohitp930 Please let me know when this PR is ready for review. Thanks

**Comment by mohitp930** [comment]
@stevemunk Strange, on my end the issue was all of the content being deleted from the article, and I couldn't track down where that happened. This likely isn't best practice, but I resolved it by copying the content from the main repo. I suppose the pr is ready for review then. Let me know what else you may need from me. Thanks!

**Pull request merged**